packages/auth-providers-api/src/dbAuth/shared.ts

```diff
@@ -7,18 +7,16 @@ import * as DbAuthError from './errors'
 // names.
 // Checks for cookie in headers in dev when user has generated graphiql headers
 export const extractCookie = (event: APIGatewayProxyEvent) => {
-  let cookieFromGraphiqlHeader
   if (process.env.NODE_ENV === 'development') {
     try {
-      cookieFromGraphiqlHeader = JSON.parse(event.body ?? '{}').extensions
+      const cookieFromGraphiqlHeader = JSON.parse(event.body ?? '{}').extensions
         ?.headers?.cookie
```

Comment:
Why isn't this 

`|| extensions?.headers?.Cookie`

as well? 

Is it because that's how the setup defines the cookie -- lowercase?

Reply:
I'm not sure about the cli side. If user setup graphiql with setup script as [official doc](https://redwoodjs.com/docs/cli-commands#setup-graphiql-headers), it should be `cookie` not `Cookie`. Below are the return value of setup script. But to be safe, I'll include `Cookie` as well.

https://github.com/redwoodjs/redwood/blob/a7b811141b201b544530354f815200e19027863c/packages/cli/src/commands/setup/graphiql/graphiql.js#L51-L55